<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSaveOptions</class>
 <widget class="QWidget" name="ImageSaveOptions">
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="saveLocationLabel">
     <property name="text">
      <string>Save Location:</string>
     </property>
     <property name="buddy">
      <cstring>kcfg_imageSaveLocation</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KUrlRequester" name="kcfg_imageSaveLocation">
     <property name="mode">
      <set>KFile::Directory|KFile::LocalOnly</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="qualityLabel">
     <property name="text">
      <string>Compression Quality:</string>
     </property>
     <property name="buddy">
      <cstring>qualitySpinner</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="qualityHLayout">
     <item>
      <layout class="QVBoxLayout" name="qualityVLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QSlider" name="kcfg_imageCompressionQuality">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QLabel" name="imageCompressionQualityHelpLable">
         <property name="text">
          <string>For lossy image formats like JPEG</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QSpinBox" name="qualitySpinner">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="filenameLabel">
     <property name="text">
      <string>Filename:</string>
     </property>
     <property name="buddy">
      <cstring>kcfg_imageFilenameFormat</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="saveLayout">
     <item>
      <widget class="QLineEdit" name="kcfg_imageFilenameTemplate">
       <property name="placeholderText">
        <string notr="true">%d</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="kcfg_preferredImageFormat">
       <property name="kcfg_property" stdset="0">
        <!-- This is not a user-visible string but a property name. Don't translate it. -->
        <string notr="true">currentText</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="PreviewLabel">
     <property name="text">
      <string comment="Preview of the user configured filename">Preview:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="preview">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="captureInstructionLabel">
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_imageCompressionQuality</sender>
   <signal>valueChanged(int)</signal>
   <receiver>qualitySpinner</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qualitySpinner</sender>
   <signal>valueChanged(int)</signal>
   <receiver>kcfg_imageCompressionQuality</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
